packages/editor/src/utils/CmsTooltip.test.ts

```diff
@@ -0,0 +1,53 @@
+import { describe } from 'vitest';
+import type { ContentObject } from '@axonivy/form-editor-protocol';
+import { findCmsEntries } from './CmsTooltip';
+
+describe('test tooltips', () => {
+  test('test cms entry search', async () => {
+    const cmsTree: Array<ContentObject> = [
+      {
+        name: 'form-test-project',
+        fullPath: '/',
+        type: 'FOLDER',
+        values: {},
+        children: [
+          {
+            name: 'greetings',
+            fullPath: '/greetings',
+            type: 'FOLDER',
+            values: {},
+            children: [
+              {
+                name: 'hello',
+                fullPath: '/greetings/hello',
+                type: 'STRING',
+                values: { en: 'Hello!' },
+                children: []
+              },
+              {
+                name: 'hello',
+                fullPath: '/greetings/longEntry',
+                type: 'STRING',
+                values: {
+                  en: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent aliquam'
+                },
+                children: []
+              }
+            ]
+          }
+        ]
+      }
+    ];
+
+    const cmsValidPath = '/greetings/hello';
+    const cmsInvalidPath = '/notfound';
+    const cmsLongPath = '/greetings/longEntry';
+    const cmsValidEntries = findCmsEntries(cmsValidPath, cmsTree);
+    const cmsInvalidEntries = findCmsEntries(cmsInvalidPath, cmsTree);
+    const cmsLongEntries = findCmsEntries(cmsLongPath, cmsTree);
+
+    expect(cmsValidEntries).toContain('en: Hello!');
+    expect(cmsInvalidEntries).toContain('no cms entry found');
+    expect(cmsLongEntries).toContain('en: Lorem ipsum dolor sit amet, consectetur ...');
```

Comment:
why this seperate variable thing? I would just do all in one line...

```suggestion
    expect(findCmsEntries('/greetings/hello', cmsTree)).toContain('en: Hello!');
    expect(findCmsEntries('/notfound', cmsTree)).toContain('no cms entry found');
    expect(findCmsEntries('/greetings/longEntry', cmsTree)).toContain('en: Lorem ipsum dolor sit amet, consectetur ...');
```